fhir-search/src/main/java/com/ibm/fhir/search/parameters/InclusionParameter.java

```diff
@@ -15,12 +17,17 @@
     private String joinResourceType;
     private String searchParameter;
     private String searchParameterTargetType;
+    private Modifier modifier;
+    private boolean userSpecifiedTargetType;
 
-    public InclusionParameter(String joinRt, String searchParm, String searchParmTargetType) {
+    public InclusionParameter(String joinRt, String searchParm, String searchParmTargetType, Modifier modifier,
+        boolean userSpecifiedTargetType) {
```

Comment:
style comment:  I try to ensure we double-indent multi-line method signatures to offset from the first line of the method:
```suggestion
            boolean userSpecifiedTargetType) {
```

Reply:
Done.